src/Storefront/Resources/views/storefront/component/product/card/box-standard.html.twig

```diff
@@ -63,7 +63,7 @@
                     {% block component_product_box_info %}
                         <div class="product-info">
                             {% block component_product_box_rating %}
-                                {% if shopware.config.detail.showReviews %}
```

Comment:
Not too sure about that, but I guess it would be nice to also remove the default configuration "showReviews", since it has no use now anymore.
`Shopware\Storefront\Framework\Twig\TemplateDataExtension#L147` that is.

@mitelg What do you think?

Reply:
or the new config must have this key